.circleci/config.yml

```diff
@@ -13,23 +13,35 @@ jobs:
     steps:
       - checkout
       - run: 
-          name: Install scripts dependencies
-          command: pip3 install -r scripts/requirements.txt
+          name: Create virtual environment
+          command: |
+            python3 -m venv venv/
+            source venv/bin/activate
+            pip3 install -r Connector/requirements.txt
+            pip3 install -r scripts/requirements.txt
       - run: 
           name: Check Lint
-          command: flake8 --statistics
+          command: |
+            source venv/bin/activate
+            flake8 --statistics
       - run: 
-          name: Deploy Development ETH Connector
+          name: Deploy ETH development nodes
           command: |
             docker-compose -f Connector/tests/docker-compose/eth.yml -p eth_development_api up --build -d
-      - run: 
+      - run:
+          name: Add ethereumgo to /etc/hosts
+          command: |
+            echo "127.0.0.1 ethereumgo" | sudo tee -a /etc/hosts
+            cat /etc/hosts
```

Comment:
Is this line showed as logs?

Reply:
Line 34 is mandatory, line 35 just for logging